The say function now checks time delay

Psyclone150 opened the pull request:
Fixes issue #15. The say function will now only display a message if a certain amount of
time has elapsed since the previous message was displayed. The default
time delay is 500ms and in choose value mode the time delay is 20ms.

Psyclone150 committed:
The say function now checks time delay

The say function will now only display a message if a certain amount of
time has elapsed since the previous message was displayed. The default
time delay is 500ms and in choose value mode the time delay is 20ms.

LauraneS commented:
This is not working as it should - nothing has changed, we still need to release the key to hear the value we're at. 
However, I don't think it is that big of an issue, if maybe the range we were choosing from was announced as we get into the node (e.g. "Pick a value between 60 & 100").

tjvr commented:
> if maybe the range we were choosing from was announced

That's covered in #17.

Psyclone150 committed:
Second attempt at introducing time delays

When scrolling fast, messages should only be displayed if a certain
amount of time has passed since the last one was displayed.
Is this what was actually asked for? Or should it actually wait to
change nodes so that no messages are skipped but scrolling is slower?

RusticFlare commented:
I don't like having the say() called on a key release because then it works very poorly for a visual user. If I hold down the key when going through a value then it jumps in random steps.
